workers/unity/Assets/Fps/Scripts/Config/MapBuilder.cs

```diff
@@ -477,7 +491,17 @@ public static string GetWorldSizeFlag(Connection connection)
                 levelInstance.transform.rotation = workerTransform.rotation;
 
                 var mapBuilder = new MapBuilder(mapBuilderSettings, levelInstance);
+
+                var volumesPrefab = mapBuilderSettings.WorldTileVolumes == null
+                    ? null
+                    : MonoBehaviour.Instantiate(mapBuilderSettings.WorldTileVolumes);
```

Comment:
Why `MonoBehaviour.Instantiate` and `MonoBehaviour.Destroy` instead of `GameObject.Instantiate` and `GameObject.Destroy` ?

Reply:
No reason; should they be `GameObject.` instead?

Reply:
Not sure - I've never seen those APIs. You may actually want `Object.Destroy()` as that's the base implementation

Reply:
typically you'd be calling Destroy or Instantiate from within a Monobehaviour script. We could change it to Object, but I don't understand why one is better than the other?

Reply:
Just consistency across the codebase really